packages/fluentui/react-northstar/src/components/Menu/Menu.tsx

```diff
@@ -256,6 +265,22 @@ export const Menu = compose<'ul', MenuProps, MenuStylesProps, {}, {}>(
       });
     };
 
+    const childProps: MenuContextValue = {
+      activeIndex: +activeIndex,
+      onItemClick: handleClick,
+      variables,
```

Comment:
This is a bit scary: as these props are related to design and can exist only under `compose`...